extensions/wikia/UserProfilePageV3/templates/UserProfilePage_renderUserIdentityBox.php

```diff
@@ -189,6 +189,14 @@
 						<li><?= wfMessage( 'user-identity-box-zero-state-gender' )->escaped(); ?></li>
 					<? endif; ?>
 				<? endif; ?>
+				<? if ( !empty( $user['bio'] ) ): ?>
+					<li id="discussion-bio"><?= wfMessage( 'user-identity-bio', $user['bio'] )->plain(); ?></li>
```

Comment:
Cool, looks good. :)

As a best practice, this message should be escaped so it is safe to output as well. Since you just need to support a span tag, I'd suggest just parsing the message (and the param can be passed in raw given it's escaped elsewhere, i.e.
```php
wfMessage( 'user-identity-bio' )->rawParams( $user['bio'] )->parse();
```